tools/run/solo5-run-virtio.sh

```diff
@@ -80,7 +80,13 @@ while true; do
         ;;
     -n)
         NETIF="$2"
-        ifconfig ${NETIF} >/dev/null || die "no such network interface: ${NETIF}"
+        # Check dependencies
+        type ip &>/dev/null ||
+            type ifconfig &>/dev/null ||
```

Comment:
I'm not sure which shell(s) support `&>`, my `/bin/sh` does not (also, http://www.shellcheck.net/ complains:
```
SC2039: In POSIX sh, 'type' is undefined.
SC2039: In POSIX sh, &> is undefined.
```